validation.js

```diff
@@ -0,0 +1,21 @@
+const form = document.querySeelector('#contact-form');
+const EMAIL_CASE_SENSITIVITY_CHECK = 'Please enter the letters in your email as lower case';
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+
+  const email = form.elements.email.value;
+  const { button } = form.elements;
+
+  const errorMessage = document.createElement('p');
+  errorMessage.textAlign = 'left';
+  errorMessage.color = '#ff0000';
+  errorMessage.fontSize = '32px';
+  button.insertAdjacentElement('beforebegin', errorMessage);
+
+  if (email.value !== email.toLocaleLowerCase()) {
+    errorMessage.innerText = EMAIL_CASE_SENSITIVITY_CHECK;
+  } else {
+    form.submit();
+  }
```

Comment:
- [ ] Your code is understandable and clean but **it is not working**. It looks like **the form is still submitting even when I put the email which is in CAPITAL LETTERS.** I really don't know why this is happening. Please rework your code above and see what is wrong. Normally, we expect the form not to submit when you input the email in capital letters but rather to show the error message. 💡

Reply:
I googled a lot, StackOverflow, and different websites, till now, I will schedule a meeting with tech support about this.